citrination_client/models/tests/test_models_client.py

```diff
@@ -73,6 +73,16 @@ def test_retrain():
     assert resp == True
 
 
+@pytest.mark.skipif(environ['CITRINATION_SITE'] != "https://citrination.com",
+                    reason="Retrain tests only supported on open citrination")
+def test_retrain_non_async():
+    """
+    Test that we can trigger a retrain
+    """
+    resp = client.retrain("5909", is_async=False)
```

Comment:
Is this actually testing anything?

Reply:
Yes -- the wait happens when the flag is `False` but this is fixed to work the other way around now.